source/app/components/molecules/row-list/index.tsx

```diff
@@ -0,0 +1,41 @@
+// package
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Divider from '@mui/material/Divider';
+import Grid from '@mui/material/Grid';
+import Link from '@mui/material/Link';
+import ListItem from '@mui/material/List';
+import Typography from '@mui/material/Typography';
+
+// scripts
+import styles from './styles';
+
+export interface Props {
+  nameFolder: string;
+  link: string;
+}
+
+const RowList = ({ nameFolder, link }: Props) => (
```

Comment:
Change component name RowList by RowItemTemplate and folder too by row-item-template